Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -44,6 +45,13 @@ struct AssessmentView: View {
             .padding(.top)
             .padding(.leading, 18)
         }
+        .overlay {
+            ZStack {
+                if umlVM.showUMLFullScreen {
+                    UMLView()
+                }
+            }
```

Comment:
It already is an `.overlay`, no need for a `ZStack`.
```suggestion
            if umlVM.showUMLFullScreen {
                UMLView()
            }
```